posts/hello-microservice-deployment-part-2.md

```diff
@@ -0,0 +1,358 @@
+# Hello Microservice Deployment Part 2: Kubernetes and Google Cloud
+
+## Introduction
+
+Hi and welcome back. This is the second part of a three part series. In [part 1](http://todo) we got acquainted with Docker by building an image for a simple web app, and then running that image. We have thus far been working with [this repo](https://gitlab.com/sheena.oconnell/tutorial-timber-deploying-microservices.git).
+
+In this part we'll be taking the docker image from part 1 and getting it to run on a Kurbenetes cluster that we set up on Google cloud.
+
+In [Part 3](http://todo) we'll use Drone.io to set up a simple CI/CD pipeline. It will test our application and roll out any changes we make to the master branch of our repo.
+
+## So what is Kubernetes (K8s) for?
+
+Brace yourself, shit's about to get real...
```

Comment:
This is too informal.

Reply:
haha, ok. I was wondering what you would say about that one. Deleted

Reply:
For the explanation on k8s objects: I actually kept the theory and practical separate on purpose. Personally I prefer this style because it makes the article more useful after the first read. Eg if I want to just look at the commands used then I can jump to a practical section, pretty much all the text in the practical section is immediately relevant to what I'm trying to achieve. So the signal to noise ratio is better.

I'll try make it more clear.